OasisDEX/oasis-borrow
# Pool finder banner in Product Finder results

# [Pool finder banner in Product Finder results](https://app.shortcut.com/oazo-apps/story/11000/pool-finder-banner-in-product-finder-results)

Note: the banner looks a tiny bit different than the one in figma (paddings), but I used exactly the same banner component, as in Discover without changing it looks.
  
## Changes 👷‍♀️

- Created `useProductHubBanner` for easily extendable banner config loader in case there are different banner in the future,
- changed `AjnaPoolFinder` feature flag to be `false` by default 🤦
  
## How to test 🧪

Turn on `AjnaPoolFinder` and check if pool finder banner is available on Product Finder in borrow and earn mode.
